angel_system/object_detection/yolov8_detect.py

```diff
@@ -9,7 +9,7 @@ def predict_hands(hand_model: YOLOv8, img0: np.array, device: str, imgsz: int) -
     """
     width, height = img0.shape[:2]
     hands_preds = hand_model.predict(
-        source=img0, conf=0.1, imgsz=imgsz, device=device, verbose=False
+        source=img0, conf=0.1, device=device, verbose=False
```

Comment:
Why remove the `imgsz` parameter? I vaguely recall this was pretty important so that we are inferencing with the same resolution that the model was trained on (768 or something).

Reply:
@Purg , when this parameter is included, the model won't return any predictions (with this version of YOLOv8 which was needed to convert the model to tensorRT).  By removing this parameter, I started getting detections.  I'm looking into if there was a bug with this version.

Reply:
One of the things I observed is that, with at least TensorRT conversion amongst possibly other versions, the image size is "baked in" to the model as part of the conversion. It may be the case that we need to convert the model with the appropriate image size we would provide here. Did you already do that when you converted the model to TensorRT and are still not getting outputs? If so, we may want to detect / indicate that the model is TensorRT as opposed to not, because this parameter is pretty required for correct predictions if we don't use the TensorRT converted model, and the current form of the node is not asserting/requiring a TensorRT input.

Reply:
@Purg , the re-export with the `imgsz` parameter worked!